lldb/source/Breakpoint/Breakpoint.cpp

```diff
@@ -848,6 +850,13 @@ void Breakpoint::GetDescription(Stream *s, lldb::DescriptionLevel level,
   const size_t num_locations = GetNumLocations();
   const size_t num_resolved_locations = GetNumResolvedLocations();
 
+  // Grey out any disabled breakpoints in the list of breakpoints.
+  if (GetTarget().GetDebugger().GetUseColor())
+    s->Printf("%s",
+              IsEnabled()
+                  ? ansi::FormatAnsiTerminalCodes("${ansi.normal}").c_str()
+                  : ansi::FormatAnsiTerminalCodes("${ansi.faint}").c_str());
```

Comment:
You need to reset the color after printing, otherwise everything in the terminal printed after will be faint.  Also seems like you can simplify this by moving the check for `IsEnabled()` into the first `if`:

```
const bool print_faint = !IsEnabled() && GetTarget().GetDebugger().GetUseColor();
if (print_faint)
     s->Print(ansi::FormatAnsiTerminalCodes("${ansi.faint}"));

[...]

if (print_faint)
     s->Print(ansi::FormatAnsiTerminalCodes("${ansi.reset}"));
```

Reply:
![image](https://github.com/llvm/llvm-project/assets/21184907/0a95ccc6-bf9a-4acb-b8e0-2746d5911f15)
I also thought that I'd have to reset the colours back to normal somewhere but when I tried this for a little the colours seemed fine without explicitly placing a reset somewhere.

I can still add a reset alongside this simplified logic though.